doc/getting-started-guide/installing.rst

```diff
@@ -178,7 +170,7 @@ Performance Monitoring
 ~~~~~~~~~~~~~~~~~~~~~~
 
 A fixed-point performance monitoring of (a part of) our codes can be seen on
-`this page <https://tomaugspurger.github.io/asv-collection/xarray/>`__.
+`this page <https://pandas.pydata.org/speed/xarray/>`__.
```

Comment:
Is this still running? I didn't know about it! And it looks like it hasn't run in 18 months?

Reply:
the last date I could find was 2019-12-13 (about 15 months). That's better than the previous link (which is dead), but of course we should only link to something that's working. For reference, I copied this link from the "contributing" guide.

Reply:
Shall we remove for the moment? 

I think this was on a recent funding proposal so we'll hopefully have much more here soon...

Reply:
> I think this was on a recent funding proposal so we'll hopefully have much more here soon...

Yes, the benchmarking work is on the NASA proposal

Reply:
I hid both links. Once we have a working setup we can uncomment them again.